internal/gapicgen/generator/config.go

```diff
@@ -1401,11 +1401,12 @@ var microgenGapicConfigs = []*microgenConfig{
 		releaseLevel:          "ga",
 	},
 	{
-		inputDirectoryPath:    "google/storage/v2",
-		pkg:                   "storage",
-		importPath:            "cloud.google.com/go/storage/internal/apiv2",
-		gRPCServiceConfigPath: "storage_grpc_service_config.json",
-		apiServiceConfigPath:  "storage_v2.yaml",
-		releaseLevel:          "alpha",
+		inputDirectoryPath: "google/storage/v2",
+		pkg:                "storage",
+		importPath:         "cloud.google.com/go/storage/internal/apiv2",
+		// The retry configuration is explicitly excluded in order to disable
+		// GAPIC-level retries and allow the veneer layer to handle retries.
```

Comment:
Does gRPCServiceConfigPath ever have anything other than retry-related configs?

Reply:
Nope not for GAPICs

Reply:
Well, we don't set timeouts on most ops at the veneer layer, but I guess so.

Reply:
So then even more reason to not have default timeouts because that is non-standard behavior relative to the client today :)